### PR TITLE
Add ACs to 0009-MRKP

### PR DESCRIPTION
**Pull request by Jiajia-Cui**

### CODE REVIEW

File: protocol/0009-MRKP-mark_price.md [hunk 1]
```diff
@@ -56,4 +56,10 @@ The mark price is set to the higher / lower of the last traded price, bid/offer.
...
  ### 5. Defined as part of the product
-  The *Mark Price* may be calculated using an algorithm defined by the product -- and 'selected' by a market parameter.
\ No newline at end of file
+  The *Mark Price* may be calculated using an algorithm defined by the product -- and 'selected' by a market parameter.
+
+# Acceptance Criteria
+
+ 1. [ ] Following all of the matching of trades resulting from a single order or the acceptance of an order onto the order book, there may be a change to the Mark Price.  (<a name="0009-MKRP-001" href="#0009-MKRP-001">0009-MKRP-001</a>)
+ 2. [ ] Mark Price should be updated after end of open auction (<a name="0009-MKRP-002" href="#0009-MKRP-002">0009-MKRP-002</a>)
+ 3. [ ] Mark Price should reflect the right decimal place setting (market decimal) (<a name="0009-MKRP-003" href="#0009-MKRP-003">0009-MKRP-003</a>)
```
Comment (TomMcL) [inline protocol/0009-MRKP-mark_price.md]: Should these be more specific about in what way the mark price should change/what it should be? (Also, it seems like there are already another set of ACs near the top of the file?)

Comment (Jiajia-Cui) [reply protocol/0009-MRKP-mark_price.md]: oh yes, I did not see them :) will close this PR for now :)

**Review by TomMcL — APPROVED** [review]
lgtm

**Comment by Jiajia-Cui** [comment]
Thanks

**Pull request merged**